src/ingest_llm_as/processors/conversation_ingestor.py

```
@@ -76,52 +80,73 @@ async def process_pending_conversations(self) -> int:
+                    # Mark as processed with error to avoid infinite loop
+                    await conn.execute("""
+                        UPDATE raw_conversations 
+                        SET processed = TRUE, 
+                            processed_at = NOW(),
+                            processing_attempts = processing_attempts + 1,
+                            last_error = $2
+                        WHERE id = $1
+                    """, record_id, f"OmegaKG Rejection: {error_msg}")
+                    return 1
+                    
+            except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)